Minor copy edits (#2415)
vuejs/vuejs.org: src/v2/guide/plugins.md

Plugins usually add global-level functionality to Vue. There is no strictly defined scope for a plugin - there are typically several types of plugins:

> 1. Add some global methods or properties e.g. [vue-custom-element](https://github.com/karol-f/vue-custom-element)

Reviewer: I like the fact that we're removing periods since it's a list item and only consists of one sentence. However, we do need to wrap the `e.g.` portion in a parentheses as it is an optional part of the sentence.

```
1. Add some global methods or properties (e.g. [vue-custom-element](https://github.com/karol-f/vue-custom-element))
```

> 1. Add some global methods or properties. (Example: [vue-custom-element](https://github.com/karol-f/vue-custom-element))

Reviewer 2: ```suggestion
1. Add some global methods or properties e.g. [vue-custom-element](https://github.com/karol-f/vue-custom-element)
```

> 2. Add one or more global assets: directives/filters/transitions etc e.g. [vue-touch](https://github.com/vuejs/vue-touch)

Reviewer: Same comment for `e.g.`

> 2. Add one or more global assets: directives, filters, transitions, etc. (Example: [vue-touch](https://github.com/vuejs/vue-touch))

Reviewer 2: ```suggestion
2. Add one or more global assets: directives/filters/transitions etc e.g. [vue-touch](https://github.com/vuejs/vue-touch)
```

> 3. Add some component options by global mixin e.g. [vue-router](https://github.com/vuejs/vue-router)

Reviewer: Same comment for `e.g.`

> 3. Add some component options by global mixin. (Example: [vue-router](https://github.com/vuejs/vue-router))

Reviewer 2: ```suggestion
3. Add some component options by global mixin e.g. [vue-router](https://github.com/vuejs/vue-router)
```

> 4. Add some Vue instance methods by attaching them to `Vue.prototype`.

Reviewer: If we're going to remove periods from all of the list items, then we should remove it here as well

```
4. Add some Vue instance methods by attaching them to `Vue.prototype`
```

> 5. A library that provides an API of its own, while at the same time injecting some combination of the above e.g. [vue-router](https://github.com/vuejs/vue-router)

Reviewer: Same comment for `e.g.`

> 5. A library that provides an API of its own, while at the same time injecting some combination of the above. (Example: [vue-router](https://github.com/vuejs/vue-router))

Reviewer 2: ```suggestion
5. A library that provides an API of its own, while at the same time injecting some combination of the above e.g. [vue-router](https://github.com/vuejs/vue-router)
```
